src/components/footer/Footer.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
 import React from 'react';
 import './Footer.css';
 
```

Comment:
Great use of rel="noopener noreferrer"!!

Reply:
Thank thank thank! Big ups to chromes lighthouse feature for teaching my that trick.

Reply:
I didn't know you could run lighthouse directly from dev tools, that's awesome!